### ISSUE TITLE
Struggles to boot

### ISSUE THREAD

**Issue by Duffmeister007** [TeamShinkansen/Hakchi2-CE#129]
Not sure whether anyone else is having this issue which seems to have affected my snes mini since the early days of the pre CE versions. Essentially on first boot the system works great but when I reset the machine it struggles to boot back up and I need to remove the power cable (switching it on and off) to get it working again. Eventually it does work but it is a bit of a pain as it probably takes about 10 minutes of trying to get the menu system to boot back up.

I have done a full uninstall, factory reset, tried flashing my original kernel from when I first installed Hackchi and have even done a factory reset / delete save states through the Snes minis own menu. However, when I reinstall everything including the new Hackchi CE version I get the same issue. Is this a common issues?

**Comment by ZeroSuf3r** [comment]
Does it happen on playing game? If yes, i think it's related to fact that "save state" doesn't work on game probably. I had several games which did same thing, hitting reset combination on gamepad or machine result in system restart. I didn't investigate any further, just deleted games. 

If it doesn't boot on first system start, then it's another issue related to your settings or your system setup. (Just guessing)

**Comment by Duffmeister007** [comment]
Thanks for the response,

I figured it out. It was actually a hardware issue. I was using a HDMI 4 way adapter so that I did not have to constantly switch HDMI cables every time I wanted to use the SNES. This was taking a little bit of power from the machine which was enough to stop it booting. Now I have stopped using this it is working a dream. 

Thought I would respond so everyone else is aware of this.
